compiler/plugins/target/LLVMCPU/LLVMCPUTarget.cpp

```diff
@@ -515,6 +517,24 @@ class LLVMCPUTargetBackend final : public TargetBackend {
                         variantOp.getName(), ".codegen.bc", *llvmModule);
     }
 
+    {
+      const char* codegenModuleFileEnv = getenv("CODEGEN_MODULE_FILE");
+      if (codegenModuleFileEnv) {
+        auto codegenModuleBuffer = llvm::MemoryBuffer::getFile(codegenModuleFileEnv);
+        if (!codegenModuleBuffer) {
+          fprintf(stderr, "FATAL: failed to open %s\n", codegenModuleFileEnv);
```

Comment:
we also don't want to be doing this in the compiler - I think this all should probably be deleted unless converted to something better built to be in a compiler

Reply:
Accidentally committed, removed in latest force push